### PR TITLE
Implement README update when BOM is updated.

### PR DESCRIPTION
**Pull request by metacosm**

### CODE REVIEW

**Comment by geoand** [comment]
This definitely passes the eye test :)

To fix the build, you should probably run `npm run lint:fix`. Note that linting can't fix all issues, you might have to manually fix some

**Comment by metacosm** [comment]
Not sure how to deal with `package-lock.json`, though… I see some versions have been changed but I don't know the whole deal around it to figure out whether that's correct or not…

File: test/support/transform/bom/updateBomFromUpstreamTest.ts [hunk 1]
```diff
@@ -99,7 +107,98 @@ const Bom = `<?xml version="1.0" encoding="UTF-8"?>
...
+        <groupId>org.apache.tomcat.embed</groupId>
+        <artifactId>tomcat-embed-core</artifactId>
+        <version>\${tomcat.version}</version>
+      </dependency>
     </dependencies>
   </dependencyManagement>
 </project>
 `;
+
+const InitialReadme = `// spring-boot
```
Comment (geoand) [inline test/support/transform/bom/updateBomFromUpstreamTest.ts]: Perhaps we could have this as a function that takes a few params and also use for UpdatedReadme?

Comment (metacosm) [reply test/support/transform/bom/updateBomFromUpstreamTest.ts]: Perhaps, though there are formatting differences between the initial and updated versions (not on purpose but still interesting to know).

**Comment by geoand** [comment]
In that case I see no reason to overthink this.

Looks great to me

**Comment by metacosm** [comment]
Should we merge it, then?

**Comment by geoand** [comment]
I vote Yes!

**Pull request merged**